components/Blog.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import Image from 'next/image';
+import { Blog1, Blog2, Blog3 } from '../assets';
+
+const Blog = () => {
+  return (
+    <section className="px-4 py-16 mx-auto sm:max-w-xl md:max-w-full lg:max-w-screen-xl md:px-24 lg:px-8 lg:py-20">
+      <div className="max-w-xl mb-10 md:mx-auto sm:text-center lg:max-w-2xl md:mb-12">
+        <h2 className="max-w-lg mb-6 font-sans text-3xl font-bold leading-none tracking-tight text-gray-900 sm:text-4xl md:mx-auto">
+          <span className="relative">Articles & Guide</span>
+        </h2>
+        <p className="text-base text-gray-700 md:text-lg">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Elementum et accumsan
+        </p>
+      </div>
+      <div className="grid gap-6 row-gap-5 mb-8 lg:grid-cols-3 sm:row-gap-6 sm:grid-cols-2">
+        <a href="#" aria-label="View Item">
```

Comment:
Instead of repeating the contents here, can we place all the data in the data folder (as I showed previously) and load all the related images and content from blog from the data folder.

To provide more context, 
- we should create a data folder
- we can create a `blog.js` file
- Inside that file, we can import the 3 Blog images add them to an array
- We will also add the pseudo content to the same array declared in the above step
- We will export the array and import it in this file
- We will create a BlogCard component to display a single blog card (the same code here from line 17 to line 28.
- We will map through the blogContent array and use the BlogCard to generate the 3 blog cards you did here.

Reply:
Noted 📝 I will Fix this